src/locale/en.json

```diff
@@ -67,7 +67,27 @@
 		}
 	},
 
-	"blended": "Blended",
+	"module": {
```

Comment:
Why do we use 2 different formats for i18n keys?

Reply:
are you referring to adding structure to `en.json` rather than having everything on the same level? IMO it's more readable and maintainable.

Reply:
Some keys use dots and some use underscores. If you have a strong opinion about how it should be done why not raise it with the team and make it consistent?